### ISSUE TITLE
Define unacceptable postmortems

### ISSUE THREAD

**Issue by Jacob-Allebach** [Jacob-Allebach/code-of-conduct#2]
- Students should not only describe successes and omit failures
- Students should not direct the blame for failures on any particular person
- Students should not write vaguely about failures or mention an issue without going into detail

**Comment by DlHolland** [comment]
Looks like you may want to add periods at the end for the formatting.  Other than that it looks good to me!

**Comment by Jacob-Allebach** [comment]
- Students should not only describe successes and omit failures.
- Students should not direct the blame for failures on any particular person.
- Students should not write vaguely about failures or mention an issue without going into detail.

**Comment by Ochirsaikhan** [comment]
When I read the first unacceptable postmortem, I can't help but reading it as "Students should NOT ONLY describe successes..., BUT ALSO." So, consider editing the sentence a little bit.

**Comment by Jacob-Allebach** [comment]
- Students should avoid omitting failures.
- Students should not direct the blame for failures on any particular person.
- Students should not write vaguely about failures or mention an issue without going into detail.

**Comment by Ochirsaikhan** [comment]
Looks good! 😎

**Comment by Ochirsaikhan** [comment]
Hey Jacob, can you make a pull request and merge your branch before the deadline?

**Comment by Jacob-Allebach** [comment]
Yep, I sent the pull request to our fork now. Just gotta get the 2 checks before it gets merged.
